public/sketch.js

```diff
@@ -156,3 +163,18 @@ function usernameChanged() {
     }
   });
 }
+
+
+//Make noise as the key pressed
+function setTone(){
+  osc = new p5.Oscillator(); 
```

Comment:
@kaichehung does `osc` here need to be a global variable?

Reply:
i.e. define `var osc;` before the `function setup()` method.

Reply:
oops! Anthony you are right! it should!
It works, so I don't even find there's a bug.

https://p5js.org/reference/#/p5.Oscillator

Reply:
thanks for looking! haha it shouldn't work but i'll trust you and merge this now. if we run into problems later will be an easy fix.